manifests/init.pp

```diff
@@ -31,13 +54,13 @@
     include 'authselect::package'
   }
 
-  if $profile_manage and $package_ensure != 'absent' {
```

Comment:
Since we added the `defined` checked, this should change probably should be reverted.